pages/common/clamdscan.md

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
I think we can update the above link to https://docs.clamav.net/manual/Usage/Scanning.html#clamdscan .

Reply:
Fixed.